internal/pkg/cli/pipeline_init.go

```diff
@@ -42,6 +43,7 @@ type InitPipelineOpts struct {
 	// Interfaces to interact with dependencies.
 	manifestWriter archer.ManifestIO
 	secretsmanager archer.SecretsManager
+	box            packd.Box
```

Comment:
Is this being used anywhere?

Reply:
Gonna use in my upcoming PR! Sorry I shouldn't have included it